### ISSUE TITLE
checkTypes.assert(false, ...) throwing wrong exception

### ISSUE THREAD

**Issue by stefanheine** [philbooth/check-types.js#78]
I found a regression in 7.1.3:

```
const checkTypes = require('check-types');

checkTypes.assert(false, 'this is false');
```

I have expected this to throw 'this is false', but in fact it's throwing 'TypeError: ErrorType is not a constructor'. this had been working as expected in 7.0.1.

**Comment by philbooth** [comment]
*Sigh*, sorry yet again!

I thought I had them all covered now, but no. I'll add tests and fix this later on.

**Comment by philbooth** [comment]
@stefanheine, I've published a fix in `7.1.4`. Let me know if you get any (more) problems! 😄

**Comment by stefanheine** [comment]
thanks!
